docs/flink-connector/flink-catalog-paimon.md

```diff
@@ -32,27 +33,26 @@ Supports most DDL and DML operations in Flink SQL, except such operations:
 
 * Paimon 0.8
 
-Higher version like 0.9 or above may also supported but have not been tested fully.
+Higher version like 0.9 or above may also support but have not been tested fully.
 
 ## Getting Started
 
 ### Prerequisites
 
 Place the following JAR files in the lib directory of your Flink installation:
 
-* paimon-flink-1.18-0.8.2.jar
-
-* gravitino-flink-connector-runtime-\${flinkMajorVersion}_$scalaVersion.jar
+- `paimon-flink-1.18-{paimon-version}.jar`
```

Comment:
Do you want to add `$` here also like the below line.

Reply:
updated